maketest.sh

```diff
@@ -29,3 +35,29 @@ else
 	$MAKE $@
 fi
 
+# For now (while this code is being prototyped) I only want this to be
```

Comment:
Could mention this is temporary location for this code that cleans up stray processes, & potentially put a reference to https://github.com/AdoptOpenJDK/TKG/issues/45 where it will live in a future iteration.